hazelcast/src/main/java/com/hazelcast/sql/impl/exec/sort/SortKey.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql.impl.exec.sort;
+
+import java.util.List;
+
+/**
+ * Single sort key.
+ */
+public class SortKey {
+    /** The key */
+    private final List<Object> key;
```

Comment:
Minor: perhaps `Object[]` could be a slightly better choice here because there could be plenty of `SortKey` created leading to big memory consumption.

Reply:
Addressed in the https://github.com/hazelcast/hazelcast/pull/18013/commits/c07a1f12034c07c68a2e6d97e0a70977ff6add89